features/FEATURE_EXPERIMENTAL_API/FEATURE_PSA/TARGET_TFM/include/psa_manifest/sid.h

```diff
@@ -130,6 +132,24 @@ extern "C" {
 #define TFM_SST_TEST_PREPARE_SID                                   (0x0000F0C0U)
 #define TFM_SST_TEST_PREPARE_VERSION                               (1U)
 
+/******** TFM_SP_PSOC_SERVICE_TEST ********/
+#define PSOC_SERVICE_TEST_PSA_INIT_WD_SID                          (0x00002001U)
+#define PSOC_SERVICE_TEST_PSA_INIT_WD_VERSION                      (1U)
+#define PSOC_SERVICE_TEST_PSA_POKE_WD_SID                          (0x00002002U)
+#define PSOC_SERVICE_TEST_PSA_POKE_WD_VERSION                      (1U)
+#define PSOC_SERVICE_TEST_PSA_KILL_WD_SID                          (0x00002003U)
+#define PSOC_SERVICE_TEST_PSA_KILL_WD_VERSION                      (1U)
+#define PSOC_SERVICE_TEST_PSA_WRITE_FLASH_SID                      (0x00002004U)
+#define PSOC_SERVICE_TEST_PSA_WRITE_FLASH_VERSION                  (1U)
+#define PSOC_SERVICE_TEST_PSA_READ_FLASH_SID                       (0x00002005U)
+#define PSOC_SERVICE_TEST_PSA_READ_FLASH_VERSION                   (1U)
+#define PSOC_SERVICE_TEST_PSA_GET_REBOOT_REASON_SID                (0x00002006U)
+#define PSOC_SERVICE_TEST_PSA_GET_REBOOT_REASON_VERSION            (1U)
+
+/******** TFM_PSOC_CLIENT_TEST ********/
+#define PSOC_CLIENT_TEST_LVL2_SID                                  (0x00002000U)
+#define PSOC_CLIENT_TEST_LVL2_VERSION                              (1U)
```

Comment:
Do we want to always include these tests in the production TF-M binary we ship with Mbed OS?

Reply:
Those test ID only take effective when the test service is enabled. In release build, test service is not enabled.

Reply:
Is the binary added by this PR the release build or the test build?

Reply:
Release Core Config.  We shouldn't be using Debug or Regression builds.

Reply:
When mbed-os-tf-m-regression-tests is used to add target support to Mbed OS with the release core config (ConfigCoreIPC.cmake), it will avoid adding these extra test services to the header here (because TF-M doesn't put these services into this header because the binary doesn't have these services available).